### PR TITLE
Declare a return value of `Uint8List` in `BytesBuilder` methods

### PR DESCRIPTION
**Pull request by tvolkert**
Part of the rollout of https://github.com/dart-lang/sdk/issues/36900

### CODE REVIEW

**Comment by tvolkert** [comment]
@natebosch @grouma @kevmoo not sure who the right reviewer / committer is for this 😄

**Comment by kevmoo** [comment]
LGTM – @natebosch ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 1.0.13
+
+* Update `BytesBuilder.takeBytes()` and `BytesBuilder.toBytes()`
```
Comment (natebosch) [inline CHANGELOG.md]: These don't seem to be visible so shouldn't impact users of this package. I'd maybe phrase this as "Internal changes for consistency with the SDK."

Comment (tvolkert) [reply CHANGELOG.md]: Done

**Comment by tvolkert** [comment]
PTAL

**Pull request merged**